pkg/sidecar-controller/snapshot_controller.go

```diff
@@ -315,7 +315,13 @@ func (ctrl *csiSnapshotSideCarController) checkandUpdateContentStatusOperation(c
 		}
 		return updatedContent, nil
 	}
-	return ctrl.createSnapshotWrapper(content)
+
+	_, groupSnapshotMember := content.Labels[utils.VolumeGroupSnapshotHandleLabel]
+	if !groupSnapshotMember {
+		return ctrl.createSnapshotWrapper(content)
+	}
+
+	return content, nil
```

Comment:
Should we have some tests for this? For now - unit tests will be good. In long term, we should have some e2e as well I think. I am not sure if all users will have a default snapshotclass when testing volumegroupsnapshot feature.

Reply:
Opened an issue to track this: https://github.com/kubernetes-csi/external-snapshotter/issues/1211